src/SearchResults.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import moment from "moment";
+
+const SearchResults = props => {
+  return (
+    <table class="table">
+      <thead class="thead-dark">
+        <tr>
+          <th scope="col">ID</th>
+          <th scope="col">Title</th>
+          <th scope="col">First Name</th>
+          <th scope="col">Surname</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room ID</th>
+          <th scope="col">Check in date</th>
+          <th scope="col">Check out date</th>
+          <th scope="col">Nights</th>
+        </tr>
+      </thead>
+      {props.results.map(booking => {
+        return (
+          <tbody>
```

Comment:
Having your `<tbody>` element inside the `return` statement of your `map` means that every row in the table has a `<tbody>` element wrapped around it. A `<table>` element should only have one `<tbody>`.